data/human/free worlds intro.txt

```diff
@@ -1327,7 +1327,7 @@ mission "FW Escort 1"
 				accept
 	
 	on accept
-		log "Serving as an escort for a Free Worlds supply convoy to Tarazed. It sounds like pirates have been specifically targeting their convoys, recently."
+		log "Serving as an escort for a Free Worlds supply convoy to Tarazed. It sounds like pirates have been blatantly targeting their convoys recently."
```

Comment:
I agree that this needs to be changed, but I'm not sure blatantly captures the right meaning. You could just remove the word altogether, or find one that fits better.

Reply:
I'm not sure this is needed. The Pirates could very well be specifically targeting them.

Reply:
> I'm not sure this is needed. The Pirates could very well be specifically targeting them.

They are specifically targeting them, except blatantly seems like a better choice of word than specifically.

Reply:
"Have specifically been targeting their convoy recently" would be the better phrasing here. While blatantly does have its own use case, specifically would imply something more coordinated, say, as revenge for banding together and shooting at them.

Reply:
There's a lot more intentionality behind the word "specifically" as opposed to "blatantly." "Specifically" suggests to me that Free Worlds convoys are being targeted to the exclusion of other potential targets. "Blatantly" doesn't hold that same weight.